db/seeds.rb

```diff
@@ -9,6 +9,7 @@
 User.destroy_all
 Merchant.destroy_all
 Item.destroy_all
+Merchant.destroy_all
 
```

Comment:
I don't understand why this is in 2 spots here. Could you explain this to me?

Reply:
Seconded

Reply:
oops, I forgot to delete the whole line while resolving merge conflict; we should remove line 12